superset/assets/src/dashboard/stylesheets/components/new-component.less

```diff
@@ -6,6 +6,7 @@
   padding: 16px;
   background: white;
   cursor: move;
+  font-weight: 600;
```

Comment:
why the larger font-weight? I think I prefer the original one.

Reply:
removed font-weight.